google-cloud-logging/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
```

Comment:
Looks like this is going to require a major version bump.

Reply:
Yes, this is marked as a breaking change and will be a 2.0. The underlying API also removed fields from the service and these changes should appear in the release notes.